common/pb/response_adjust_mod.proto

```diff
@@ -0,0 +1,7 @@
+import "response.proto";
+
+message Response_AdjustMod{
+	extend Response {
+		optional Response_AdjustMod ext = 1011;
```

Comment:
If I recall correctly all the proto files are a different indent than the 4 space.  CLion tells me that all the time when im working with them.